### PR TITLE
Fixed pip/setuptools installation on Jython

### PR DESCRIPTION
**Pull request by agronholm**
This clears one roadblock from getting virtualenv to work with Jython. There are more, but they should probably be fixed on Jython's side.

### CODE REVIEW

**Comment by rbtcollins** [comment]
Jython doesn't honour the PYTHONPATH variable? :(. Well it is what it is, but I think we need a comment explaining when we can remove the workaround...

**Comment by agronholm** [comment]
There already is -- I explicitly mentioned Jython 2.x. I just committed a change in the Jython 3 repository that makes Jython use PYTHONPATH instead.

File: virtualenv.py [hunk 1]
```diff
@@ -955,6 +955,7 @@ def install_wheel(project_names, py_executable, search_dirs=None):
         call_subprocess(cmd, show_stdout=False,
             extra_env = {
                 'PYTHONPATH': pythonpath,
+                'JYTHONPATH': pythonpath,  # for Jython 2.x
```
Comment (rbtcollins) [inline virtualenv.py]: Cool - so here please say 

 # For Jython < 3.x

instead. Thanks!

**Comment by agronholm** [comment]
I've updated the PR.

**Comment by agronholm** [comment]
So where do we stand with this?

**Comment by agronholm** [comment]
This is the ONLY thing remaining that prevents virtualenv from working properly on Jython 2.7. Please merge this now!

**Comment by rbtcollins** [comment]
So, it looked fine with that tweak to me, but I'm not a committer - sorry.

**Pull request merged**